src/components/SearchBar.vue

```diff
@@ -92,7 +92,7 @@ export default {
       },
     },
     type: {
-      get: () => store.state.query.type,
+      get: () => store.getters['query/filters/typeFilter'].value,
       set(newType) {
         if (this.type !== newType) {
           enterSearchQuery({ type: newType });
```

Comment:
Are we going to have code *exactly* like this for every filter we're adding?

In that case, I would suggest that *as soon as* we agree on the API, we abstract this somehow. Doesn't have to be part of the PR but an issue should be made *before* merging to keep track of it.

Reply:
no, this is just the type filter. The type filter is the only non-regular filter. For the other filters you just have to add a json definition.